packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_querying_shadow.spec.ts

```diff
@@ -0,0 +1,13 @@
+// @ts-ignore / session support is needed for visiting about:blank between tests
+context('multi-domain shadow dom', { experimentalSessionSupport: true, experimentalMultiDomain: true }, () => {
+  beforeEach(() => {
+    cy.visit('/fixtures/multi-domain.html')
+    cy.get('a[data-cy="shadow-dom-link"]').click()
+  })
+
+  it('types into input', () => {
```

Comment:
There is an implicit assertion here that there is a shadow dom that has an input element (i.e. it will fail the test if this is not true) but I went ahead and added an explicit assertion as well.

Reply:
Updated [here](https://github.com/cypress-io/cypress/pull/19734/commits/bbabb3e7b8be84eb2d7995af821eff037eb9d3d9)